index.html

```diff
@@ -11,9 +11,23 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+<script src="script.js" defer></script>
+<script src="contributors.js" defer></script>
+<link rel="stylesheet" href="pop-up-menu.css">
 </head>
-<body>
-    <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i></header>
+<body id="about-page">
+    <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i>
+        <div id="pop-up-menu-wrapper">
+            <i class="fa-solid fa-xmark" id="cancel"></i>
+        <div class="pop-up-menu">
+            <ul class=".pop-up-list">
+                <li class="pop-up-li"><a class="page-link" href="home-page.html">Home</a></li>
+                <li class="pop-up-li"><a class="page-link" href="index.html">About</a></li>
```

Comment:
- [ ] In your about page in the desktop version also it would be great if you fix spaces here so please decrease the margin/padding between the elements since your design it should be identical to the original design so please fix the spaces here according to the requirements 🤩🔥♻🙏🏻✨

- Your design
![image](https://user-images.githubusercontent.com/58553711/219922349-817fb1d9-a5cb-4750-ace4-47c921e33fc4.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/219922363-9419568d-7c08-4195-ab2f-c2c3f8c28361.png)